CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+- Consider `:reader-aliases` for completion referred keywords.
```

Comment:
The wording doesn't seem very natural to me

Reply:
Agree :) It's such a specific change. I wasn't sure how to describe it well. Any suggestions? Maybe something simple, like "More robust completion for referred keywords"?

Reply:
Yeah, that'd be better. Perhaps adding an example in the changelog would be useful as well to make it clearer.

Reply:
Updated. I added a little explanation, please let me know if it's good enough.